site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -494,14 +437,25 @@ $ kubectl delete mc my-release
 
 </div>
 
+## Uninstall Milvus Operator
 
-## Delete the K8s cluster
+Run the following command to uninstall Milvus Operator.
 
-When you no longer need the K8s cluster in the testing environment, you can delete it.
+### Uninstall Milvus Operator by Helm command
 
-If you use minikube to install the K8s cluster, run `$ minikube delete`.
+```
+$ helm -n milvus-operator uninstall milvus-operator
+```
+
+### Uninstall Milvus Operator by `kubectl` command
+
+```
+$ kubectl delete -f https://raw.githubusercontent.com/milvus-io/milvus-operator/v{{var.milvus_operator_version}}/deploy/manifests/deployment.yaml
+```
+
+## Delete the K8s cluster
 
-If you use kind to install the K8s cluster, run `$ kind delete cluster --name myk8s`.
+When you no longer need the K8s cluster in the test environment, you can delete it, run `$ minikube delete` to delete it.
```

Comment:
When you no longer need the K8s cluster in the test environment, you can run `$ minikube delete` to delete it.

Reply:
[dbe9c11](https://github.com/milvus-io/milvus-docs/pull/1643/commits/dbe9c1115de0a61c607f10550716f6b5660a8a1a)